python/tk_framework_alias/server/alias_bridge.py

```diff
@@ -501,7 +501,11 @@ def __serve_app(self):
         # is single threaded (e.g. can only access the socketio server from the thread it was
         # created in).
         wsgi_logger = framework_utils.get_logger(self.__class__.__name__, "wsgi")
-        eventlet.wsgi.server(self.__server_socket, self.__app, log=wsgi_logger)
+        try:
+            eventlet.wsgi.server(self.__server_socket, self.__app, log=wsgi_logger)
+        except Exception as e:
```

Comment:
Is it possible to have a more specific catch instead?
If not, could you explain why in a small comment?

Reply:
I see other similar except in your change bellow. So I raise the same question for those too ;)

Reply:
For this case, the function we're calling `eventlet.wsgi.server` does not provide documentation on what exceptions it throws; so we want to be sure we catch any exception so that we can report it, instead of failing silently